bin/impl/commands.sh

```diff
@@ -300,7 +299,7 @@ function uno_fetch_main() {
 function uno_wipe_main() {
   local yn
   uno_kill_main
-  read -r -p "Are you sure you want to wipe '$INSTALL'? " yn
+  read -r -p "Are you sure you want to wipe '$INSTALL'? [Y/n] " yn
```

Comment:
I don't think this change is necessary. The capital `Y` implies it's the default if you just press Enter, but it isn't. No is the default. It also implies that only `Y` and `n` are acceptable responses, but spelling out "yes" also is acceptable. Is the current wording unclear about what one can enter? If not, then I'd leave this one the way it was.

Reply:
From personal experience it wasn't too clear to me how to proceed when first prompted. I wasn't aware that the capital letter implied the default. Do you think this should be removed entirely or is there a more appropriate "y/n/yes/no" variation that this could be changed to.

Reply:
I think it's less verbose to simply omit it, as it originally was. Both `y` and `yes` work, and neither are case-sensitive. So, even if the user isn't clear what options are available, the script is tolerant.